app/core/interfaces/map/interface.js

```diff
@@ -32,7 +32,8 @@ define([
       var mapOptions = {
         center: center,
         zoom: 12,
-        disableDefaultUI: this.options.settings.get('read_only') || !this.options.canWrite
+        disableDefaultUI: this.options.settings.get('read_only') || !this.options.canWrite,
+        gestureHandling: 'cooperative',
```

Comment:
Using **cooperative gesture handling** lets the user control the panning and scrolling behavior of a map when viewed on a mobile device.